src/libsyntax/print/pprust.rs

```diff
@@ -3186,10 +3186,11 @@ mod tests {
     a. ..;
     for token in program.chars()
     {
-        match token {
-                        '+' => accumulator += 1 , '-' => accumulator -= 1 , '*' => accumulator *= 2 , '/' => accumulator /= 2 , _ => {}
+        match token 
+        {
+            \'+\' => accumulator += 1 , \'-\' => accumulator -= 1 , \'*\' => accumulator *= 2 , \'/\' => accumulator /= 2 , _ => {}
```

Comment:
Two interesting things here:
* `{` should never start a new line *before* it, always *after*
 * it gets trickier sometimes thought (`}` and `;` come to be mind)
 * the whole reason there's a `pp.rs` is for automating breaks
 * but I'm not sure the algorithm is appropriate for Rust or bug-free
* `,` should never have a space *before* it, always *after*